frontend/src/components/pages/recommandationForm/RecommendationForm.js

```diff
@@ -0,0 +1,96 @@
+import styled from "styled-components/macro"
+import Step1 from "./stepcomponents/Step1"
+import Step2 from "./stepcomponents/Step2"
+import Step2B from "./stepcomponents/Step2B"
+import Step3 from "./stepcomponents/Step3"
+import Step3B from "./stepcomponents/Step3B"
+import Step4 from "./stepcomponents/Step4"
+import Step5 from "./stepcomponents/Step5"
+import { useState } from "react"
+import Summary from "./stepcomponents/Summary"
+import {useHistory} from "react-router";
+
+
+function RecommendationForm() {
+   let [formStep, setFormStep] = useState(1)
+   const [wineConfiguration, setWineConfiguration] = useState({
+      occasion: "",
+      wineStyle: "",
+      region: "",
+      alcohol: "",
+      taste: "",
+   })
+   const history=useHistory();
+
+   const nextStep = (value) => {
+      setFormStep(value + 1)
+   }
+   const handleDataInput = (name, value) => {
+      setWineConfiguration({ ...wineConfiguration, [name]: value })
+   }
+
+   const handleSubmit = () => {
+      //future PostMapping
+      console.log("Data send")
+      history.push("/recommendations")
+   }
+
+   return (
+      <FormWrapper>
+         <MainHeading>Let's find your wine</MainHeading>
+         <Form>
+
+            {
```

Comment:
Iam not happy with the conditional rendering part. Its really hard to read. It would be great ti simplify that.

Reply:
done :-)